### DISCUSSION TITLE
Doris备份导出到s3有关于endpoint的问题

### DISCUSSION THREAD

**Discussion by weakdouqing** [apache/doris#24948] [A - General / Q&A]
在我使用下列命令想要备份Doris数据库中的数据到s3上面的时候，发生错误

```sql
CREATE REPOSITORY `s3_test`
WITH S3
ON LOCATION "s3://my_bucket_name"
PROPERTIES
(
    "s3.endpoint" = "s3://arn:aws:s3:region:xxxx:accesspoint/my_bucket_name_endpoint",
    "s3.access_key" = "ak",
    "s3.secret_key"="sk",
    "s3.region" = "region"
);
```
错误信息为：
> detailMessage = Failed to create repository: errors while get file status Incorrect endpoint: hostname cannot be null

在当我去询问我的同事有关于aws endpoint相关事项的时候，他也不太清楚这一块的操作
>求助：

我该如何操作来获取到这个正确的aws s3 的endpoint

**Comment by LemonLiTree** [comment]
Which version of doris are you using?In addition, how do you usually access this s3?

**Reply by weakdouqing** [reply]
The Doris version I am currently using is 1.2.0.
When I usually access s3, I only need ak, sk and bucket.

**Reply by LemonLiTree** [reply]
Endpoint is not used?  Endpoint is the address. . .

**Reply by LemonLiTree** [reply]
like xxx.xxx.com

**Reply by weakdouqing** [reply]
I know this method, but whenever I create an endpoint, the connection given by aws s3 is of the type s3://arn:aws:s3:region:xxxx:accesspoint, so I am not sure what I am doing. What should I do? Do you have any better solutions?

**Reply by weakdouqing** [reply]
Flink版本：1.14.0
Doirs版本：1.2.0
flink-doris-connector-1.14_2.12：1.1.1

或者换一种想法，我想使用Flink读Doris一个库里面的其中一个表，现在要迁移到另一个Doris集群中，但是当我使用Flink读Doris的时候，发现Flink总是运行了一会儿就自动结束了，其中配置使用了Boundedness.CONTINUOUS_UNBOUNDED无界流
有其他更好的解决办法嘛？
